src/atomics/atomics.md

```diff
@@ -0,0 +1,121 @@
+# Atomics
+
+Rust pretty blatantly just inherits the memory model for atomics from C++20. This is not
```

Comment:
It's probably good to mention the correspondence to `atomic_ref` again, as was added to the std docs https://doc.rust-lang.org/std/sync/atomic/